Cabal/Distribution/Types/GenericPackageDescription/Lens.hs

```diff
@@ -35,51 +35,37 @@ import Distribution.Version (VersionRange)
 -------------------------------------------------------------------------------
 
 condBenchmarks :: Lens' GenericPackageDescription [(UnqualComponentName, CondTree ConfVar [Dependency] Benchmark)]
-condBenchmarks f (GenericPackageDescription x1 x2 x3 x4 x5 x6 x7 x8) = fmap (\y1 -> GenericPackageDescription x1 x2 x3 x4 x5 x6 x7 y1) (f x8)
+condBenchmarks f s = fmap (\x -> s { T.condBenchmarks = x }) (f (T.condBenchmarks s))
```

Comment:
these are auto-generated, why change?

Reply:
Oh I didn't realize that. Can they be easily regenerated or was it a one-off copy-the-th-splice?

Reply:
If it was a one-off generation, I'm inclined to keep this as more readable and immune to argument order changes.